src/glomap/scene/view_graph.h

```diff
@@ -32,6 +32,18 @@ struct ViewGraph {
   int MarkConnectedComponents(const colmap::Reconstruction& reconstruction,
                               std::unordered_map<frame_t, int>& cluster_ids,
                               int min_num_images = -1);
+
+  // Mark image pairs as invalid if their relative rotation differs from the
+  // reconstructed rotation by more than max_angle_deg.
```

Comment:
Maybe add: `// Keeps existing invalid edges as invalid.`

Same for other 2 methods.